<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoModelu</class>
 <widget class="QDialog" name="PoModelu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Po modelu</string>
  </property>
  <widget class="QLabel" name="labelModel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBoja">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Boja</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGorivo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gorivo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGodinaProizvodnje">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Godina prozvodnje</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxModel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBoja">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGorivo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrikazi">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Prikazi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzadji">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Izadji</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>280</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxGodinaProizvodnje">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2015 i novije</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2010 - 2014</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2005 - 2009</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2000 - 2004</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1995 - 1999</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1990 - 1994</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>pre 1990</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
